jobs/update-nodejs-version.js

```diff
@@ -5,7 +5,13 @@ const githubQueue = require('../lib/github-queue')
 const yaml = require('js-yaml')
 const createBranch = require('../lib/create-branch')
 const getConfig = require('../lib/get-config')
-const { getNodeVersionIndex, getNodeVersionsFromTravisYML, addNodeVersionToTravisYML } = require('../utils/utils')
+const {
+  hasNodeVersion,
+  getNodeVersionIndex,
+  getNodeVersionsFromTravisYML,
+  addNodeVersionToTravisYML,
+  addNodeVersionToNvmrc
```

Comment:
Only nitpick: Can you change `addNodeVersionToNvmrc` to `updateNodeVersionInNvmrc`? Since it only ever has one.